test/cpp/test_status_common.h

```diff
@@ -77,36 +78,22 @@ class StatusTest : public testing::TestWithParam<CppStacktracesMode> {
 
 namespace testing {
 
+// Catches the c10 error, and transforms it into a runtime error, calling
+// what() or what_without_backtrace() depending on IsShowCppStacktracesMode().
+#define XLA_THROW_RUNTIME_ERROR_FROM_C10_ERROR(block) \
```

Comment:
Remove this? Prefer straightforward test code that just catches the c10::Error and verifies it.